models/Post.js

```diff
@@ -29,10 +30,7 @@ const PostSchema = new mongoose.Schema({
     type: String,
     required: true
   },
-  comments: {
-    type: Array,
-    required: false
-  },
+  comments: [Comments.schema],
```

Comment:
This is where you can add Comments Schema as an embedded document to the Post Schema, to save yourself from additional queries after first querying for your Post. 

You automatically get the Comments associated with that Post, just by querying the Post.